src/ansi-c/library/cprover.h

```diff
@@ -35,6 +35,7 @@ __CPROVER_size_t __CPROVER_zero_string_length(const void *);
 __CPROVER_size_t __CPROVER_buffer_size(const void *);
 __CPROVER_bool __CPROVER_r_ok(const void *, __CPROVER_size_t);
 __CPROVER_bool __CPROVER_w_ok(const void *, __CPROVER_size_t);
+void __CPROVER_old(const void *);
 
```

Comment:
I would make that
`void __CPROVER_old();`
since you want to support arbitrary expressions eventually.